src/services/host_info.rs

```diff
@@ -0,0 +1,104 @@
+use std::net::{IpAddr, Ipv4Addr, Ipv6Addr};
+
+use sysinfo::System;
```

Comment:
I am not sure if `crate/psh-system` can provide the host information you need, but I think these information should be provided in `crate/psh-system` and gRPC service simply call `crate/psh-system` public functions to encapsulate data for needs.

Reply:
I will have a look